src/drive/web/modules/filelist/FileOpener.jsx

```diff
@@ -42,14 +42,9 @@ const enableTouchEvents = ev => {
     return false
   }
 
-  // remove events when they are on the file's path, because it's a different behavior
-  const parentLink = getParentLink(ev.target)
-  if (
-    parentLink &&
-    parentLink.className.indexOf(styles['fil-file-path']) >= 0
-  ) {
-    return false
-  }
+  // We need to filter here, is the event target the FileOpener or a link INSIDE the FileOpener?
+  // Assuming the FileOpener itself will never be inside an anchor, we can use the following guard
+  if (ev.srcEvent.target.closest('a')) return false
```

Comment:
I know that there is no test for this method, but I think this is something we should add. We have too many regressions on this. What do you think?

Reply:
I don't know it's hard to test, would basically needs to render a whole DOM tree up to window. I'm basing myself on the assumption that it's impossible to find an anchor parent in the event unless it was launched by a child node, ie router link in this case. I tried to not be specific but I could also use a class on the folder link. But then the bug could happen elsewhere if some link is added

Reply:
I guess FileOpener having anchor as a parent should be a bug anyway, it would not be valid HTML

Reply:
> I don't know it's hard to test, would basically need to render a whole DOM tree up to window. I'm basing myself on the assumption that it's impossible to find an anchor parent in the event unless it was launched by a child node, ie router link in this case. I tried to not be specific but I could also use a class on the folder link. But then the bug could happen elsewhere if some link is added

We already have a DOM test for /recent view. We already do some click on a few actions. I don't think it would be difficult to test with that, no?

Reply:
I guess it's possible sure, maybe verifying react-router is only invoked once. Or creating stubs and see where react-router sends me on different clicks. I'm worrying about delivery time though, my tests are failing in local and I don't know why. I need to fix this and then write the tests. I'd propose to do this in another PR if at all possible

Reply:
>  I need to fix this and then write the tests. I'd propose to do this in another PR if at all possible

Since the test file already exist, already display a File within the view with a path displayed I think, we should be able to click on the path and check the new url. I think this is mandatory.

Reply:
You can also only test this method by creating an event with different kind of targets / html .

Reply:
I added two expects (I would prefer to have multiple tests but this test is already using a pattern with a lot of assert in a single test). 

I also add a small prettier file to match the eslint config for IDE purposes, not just mine but everyone having Prettier plugin + VSCODE for instance. Is that ok?